src/ActionModal/components/ActionModal.vue

```diff
@@ -799,7 +801,10 @@ export default {
           delegatorAddress: this.session.address
         }
       },
-      update: result => result.rewards
+      update: result => result.rewards,
+      skip() {
+        this.loadRewards
```

Comment:
This looks like a clue for a potential refactor. Rewards should probably only be loaded for certain modals?